nyx_client/nyx_client/client.py

```diff
@@ -718,6 +777,16 @@ def update_data(
             data["downloadURL"] = download_url
         if license_url:
             data["licenseURL"] = license_url
+        if access_control == "none":
+            data["accessControl"] = [ALLOW_NONE]
+        if access_control == "all":
+            data["accessControl"] = [ALLOW_ALL]
+        if circles:
+            data["circles"] = [c.did for c in circles]
+
+        # We need to specify either access control or one circle
```

Comment:
(v. unimportant) I think in this case `not (circles or access_control)` would suffice (to mirrow the above "both specified") bit.
Is it worth both of these checks being in the same place? (maybe here rather than above)